uint8 ID # ID > 99 indicates ship
float64 x #x position in meters
float64 y #y position in meters
float64 theta #orientation in radians or degrees?
float32 PushingForce # Newton